eq-author/src/storybook/Components/Radio.stories.mdx

```diff
@@ -0,0 +1,101 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+import styled from "styled-components";
+
+import { Input, Label } from "components/Forms";
+
+import { colors } from "constants/theme";
+
+<Meta title="Components/Radio button" component={Input} />
+
+export const RadioLabel = styled.label`
+  padding: 1em 1.5em;
+  border-radius: 0.25em;
+  margin-bottom: 0.8em;
+  border: 1px solid ${colors.bordersLight};
+  flex: 1 1 33.3333333%;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.18);
+  color: ${colors.text};
+  position: relative;
+`;
+
+export const RadioTitle = styled.span`
+  font-weight: bold;
+  letter-spacing: 0;
+  margin-left: 3em;
+  color: ${colors.text};
+  display: flex;
+`;
+
+export const RadioDescription = styled.span`
+  font-weight: normal;
+  font-size: 1em;
+  letter-spacing: 0;
+  margin-left: 3em;
+`;
```

Comment:
Could these components be put in Author's source instead and referenced from storybook? Otherwise there's no way for people to reference these components in author (if they like what they see in the storybook code and want to use it on a page).

Could put in same dir as the radio button code itself or in `/components` so it's accessible to other pages. Then could use these components in place of the `LegalTitle` etc. on the actual page.